src/config/resolve-config-editorconfig.js

```diff
@@ -8,7 +9,15 @@ const editorConfigToPrettier = require("editorconfig-to-prettier");
 const findProjectRoot = require("find-project-root");
 
 const maybeParse = (filePath, config, parse) => {
-  const root = findProjectRoot(path.dirname(path.resolve(filePath)));
+  // findProjectRoot will throw an error if we pass a nonexistent directory to
+  // it, which is possible, for example, when the path is given via
+  // --stdin-filepath. So, first, traverse up until we find an existing
+  // directory.
+  let dirPath = path.dirname(path.resolve(filePath));
+  while (dirPath !== "/" && !fs.existsSync(dirPath)) {
```

Comment:
Does `dirPath !== "/"` work on Windows?

Reply:
Good catch! I don't think it would've worked. I switched it to instead use `path.parse(...).root`, and then compare the dirname to that. I tested the behavior by trying out `path.win32.parse` and `path.win32.dirname`, and it seems to work as expected.